thrall/app/lib/Config.scala

```diff
@@ -23,7 +23,7 @@ object Config extends CommonPlayAppConfig {
 
   val imageBucket: String = properties("s3.image.bucket")
 
-  val imagesAlias = properties("es.index.alias")
+  val writeAlias = properties("es.index.aliases.write")
```

Comment:
I actually don't think thrall _should_ touch the read alias. in the sense that, if it's query-ing to determine what to do next, it should be doing so on the write alias shouldn't it? (<- not rhetorical ^_^) 

I double checked anyway and I don't see anywhere in Thrall where a query reads w/o writing.